src/components/molecules/ReactionsBar/ReactionsBar.scss

```diff
@@ -0,0 +1,40 @@
+@import "scss/constants";
+
+$reaction-bar-indent: 5px;
```

Comment:
Any specific reason for `$reaction-bar-indent` variable while `$spacing--xs` is being used (making `1px` difference between the two)

Reply:
No specific reason, changed in https://github.com/sparkletown/sparkle/pull/1339/commits/a1a457d7ce90c4b5d4ed233c913bb479b9d16cbc